cargo-quickinstall/src/main.rs

```diff
@@ -34,11 +37,22 @@ fn get_latest_version(crate_name: &str) -> std::io::Result<String> {
             --user-agent 'cargo-quickinstall build pipeline (alsuren@gmail.com)' \
             --location \
             --fail \
-            'https://crates.io/api/v1/crates/{}' \
-            | jq -r .versions[0].num",
+            'https://crates.io/api/v1/crates/{}'",
         crate_name
     );
-    bash_stdout(&command_string)
+    match bash_stdout(&command_string) {
+        Ok(stdout) => match stdout.parse() {
+            Ok(parsed) => {
+                let parsed: JsonValue = parsed;
```

Comment:
you can probably use `stdout.parse::<JsonValue>()` above, and delete this line.

I also prefer to use the ? operator so that my "happy path" code doesn't drift to the right so much.

Reply:
Ok, I pushed some changes using the `?` operator, I don't think `stdout.parse::<JsonValue>()` is necessary here.